include/z64cutscene.h

```diff
@@ -513,4 +523,21 @@ typedef struct {
     /* 0x8 */ s16 relativeToPlayer;
 } CutsceneCameraMove; // size = 0xC
 
+#ifdef ENABLE_MOTION_BLUR
+typedef union {
+    struct {
+        /* 0x0 */ u16 type;
+        /* 0x2 */ u16 alpha;
+        /* 0x4 */ u16 startFrame;
+        /* 0x6 */ u16 endFrame;
+    };
+    s32 _words[2];
+} CsCmdMotionBlur; // size = 0x8
+
+typedef enum {
+    /* 1 */ CS_MOTION_BLUR_ENABLE = 1, // enable instantly
+    /* 2 */ CS_MOTION_BLUR_DISABLE // disable gradually
```

Comment:
These comments aren't needed anymore since both enable and disable are more similar now